org.eclipse.winery.topologymodeler.ui/TmDependencyTestApp/tslint.json

```diff
@@ -0,0 +1,142 @@
+{
+  "rulesDirectory": [
+    "node_modules/codelyzer"
+  ],
+  "rules": {
```

Comment:
Similar to the original app, you can reuse the root tslint file.

Reply:
done